intera_examples/scripts/constrained_zeroG.py

```diff
@@ -244,15 +247,22 @@ def main():
         rospy.loginfo(msg)
         pub.publish(msg)
 
+        rs = intera_interface.RobotEnable(CHECK_VERSION)
```

Comment:
What does this do? Is this something that needs to be in other scripts as well?

Reply:
This just gives me access to the robotEnable state. Only scripts that check if the robot is enable need this.

Reply:
Great - thanks!